clumio/resource_organizational_unit.go

```diff
@@ -0,0 +1,307 @@
+// Copyright 2021. Clumio, Inc.
+
+// clumio_organizational_unit definition and CRUD implementation.
+package clumio
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	orgUnits "github.com/clumio-code/clumio-go-sdk/controllers/organizational_units"
+	"github.com/clumio-code/clumio-go-sdk/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// clumioOrganizationalUnit returns the resource for Clumio AWS Connection.
+
```

Comment:
Nit: Extra newline and also `Clumio AWS Connection` -> `Clumio Organizational Unit`